csharp/ql/lib/experimental/code/csharp/Cryptography/NonCryptographicHashes.qll

```diff
@@ -10,7 +10,7 @@ import csharp
 private import DataFlow
 private import semmle.code.csharp.dataflow.TaintTracking2
 
-predicate maybeANonCryptogrphicHash(Callable callable, Variable v, Expr xor, Expr mul, LoopStmt loop) {
+predicate maybeANonCryptographicHash(Callable callable, Variable v, Expr xor, Expr mul, LoopStmt loop) {
```

Comment:
This is a publicly exposed predicate, so the predicate with the incorrect spelling needs to be deprecated and not removed directly.

Reply:
Not needed, I think, as this is in `experimental`.